src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -131,6 +134,47 @@ struct devInfo
   }
 };
 
+struct aieCoreInfo
```

Comment:
No camelCase please, please fix devInfo while at it :-)

Reply:
fixed